release-doc-in-process/all_vf.TTL

```diff
@@ -427,6 +441,47 @@ vf:resourceConformsTo
         rdfs:range      vf:ResourceSpecification ;
         rdfs:comment    "The primary resource knowledge specification or definition of an existing or potential resource." .
 
+vf:intents
+        a                   owl:ObjectProperty ;
+        rdfs:label         "intents" ;
+        rdfs:domain         vf:Proposal ;
+        rdfs:range          vf:Intent ;
+        rdfs:comment        "The primary offers, requests, estimates, or other intents proposed." .
+
+vf:reciprocalIntents
```

Comment:
In what cases we can not rely on *provider* and *recipient* relations from intents that proposal applies to? For proposals that should lead to exchange, one intent would specify agent with *provider* and another one with *recipient*. Practically in such proposals an agent 'offers' and 'requests' something at the same time.

Reply:
I don't think provider and receiver have anything to do with which intent is reciprocal (if any).  Intent software I have seen (as well as @matslats proposed layout) has a choice for is this proposal an offer or a request.  Since you did not want that, you can only tell based on what is reciprocal.  I don't know the percentages, but I think a fairly large percentage of people will not want their reciprocal intent to imply they want that intent to be matched.

Reply:
If an agent does want to offer and request something at the same time, then neither on is "reciprocal" and they don't use that.

Reply:
Can you provide an example of what you would consider a Proposal
* that references two Intents
* aims at providing a path to an Exchange Agreement
* out of those two referenced intents, one specifiers *provider* and the other one DOES NOT specify *recipient*

Just to make sure that we don't have misunderstanding here. Agent creates all the Intents first during initial phase of planning. Later on Agent creates various Proposals, where each Intent can get referenced from any number of different Proposals. So we don't have Intents specific to a Proposal, Intents just express initial plan of given agent, which later will lead to commitments and events.

Reply:
>out of those two referenced intents, one specifiers provider and the other one DOES NOT specify recipient

I don't understand the implications of this.  If they are intended to be reciprocal, then one would specify provider and the other recipient.  But that doesn't tell you which one is reciprocal.

>Just to make sure that we don't have misunderstanding here. Agent creates all the Intents first during initial phase of planning. Later on Agent creates various Proposals, where each Intent can get referenced from any number of different Proposals. So we don't have Intents specific to a Proposal, Intents just express initial plan of given agent, which later will lead to commitments and events.

I don't think there is a misunderstanding, I agree that is a valid scenario.  There is also a scenario where people just want to propose exchanges from inventory.  For example, distributors, aggregators, just individual people with something to get rid of or that they want.

Reply:
> If they are intended to be reciprocal, then one would specify provider and the other recipient. But that doesn't tell you which one is reciprocal.

I think Commitments made based on those Intens, will stay reciprocal to one another. I would see *reciprocity* as symmetric relation. Having an Exchange Agreement with two Commitments
* provider: Alice, recipient: Bob, work: 2h, under: an-agreement
* provider: Bob, recipient: Alice, use: 8h, under: an-agreement

I wouldn't see only one of them as reciprocal to the other one. Once again Commitments in Exchange Agreement seem to me mutually reciprocal.

> There is also a scenario where people just want to propose exchanges from inventory. For example, distributors, aggregators, just individual people with something to get rid of or that they want.

In that case they can create Intent to *issue* that something which references them as *provider*. Then they can reference that Intent in any number of Proposals, which if the aim at Exchange Agreement, would each also reference another Intent which references that agent as *recipient*.